src/supla/network/html/button_update.h

```diff
@@ -0,0 +1,22 @@
+#ifndef SRC_SUPLA_NETWORK_HTML_BUTTON_UPDATE_H_
+#define SRC_SUPLA_NETWORK_HTML_BUTTON_UPDATE_H_
+
+#include <supla/network/html_element.h>
+#include <supla/network/esp_web_server.h>
```

Comment:
instead of adding include to esp_web_server here, you can use forward class declaration here, and include proper header in cpp file. This practice speeds up compilation time (especially on recompilation).
Forward declaration will look like this:
```
namespace Supla {
  class EspWebServer;
}  // namespace Supla